app/src/test/java/org/simple/clinic/drugs/search/DrugSearchUiRendererTest.kt

```diff
@@ -26,4 +26,19 @@ class DrugSearchUiRendererTest {
     verify(ui).hideSearchResults()
     verifyNoMoreInteractions(ui)
   }
+
+  @Test
+  fun `when search query is not empty, then show search results`() {
+    // given
+    val searchQuery = "Amlodipine"
```

Comment:
Can we inline this variable?

Reply:
Yep, I will use the function params names. I just wanted to have a name for the variable